src/accidental.ts

```diff
@@ -50,20 +49,8 @@ export class Accidental extends Modifier {
   readonly type: string;
   /** To enable logging for this class. Set `Vex.Flow.Accidental.DEBUG` to `true`. */
   static DEBUG: boolean = false;
-  protected accidental: {
-    code: string;
-    parenRightPaddingAdjustment: number;
-  };
-  public renderOptions: {
-    parenLeftPadding: number;
-    fontScale: number;
-    parenRightPadding: number;
-  };
+  protected accidental: string;
```

Comment:
This is a SMuFL unicode glyph of the form '\uXXXX', correct? Since this approach is new to V5, maybe we should start to label the instance variables where this applies. For example, here we could put a comment:

```ts
// `accidental` is a string from the SMuFL specification.
// For example, '\uE260' is the flat symbol.
// See: `accidentalFlat` at https://w3c.github.io/smufl/latest/tables/standard-accidentals-12-edo.html
```

Reply:
I have removed the member variable and used `this.type` directly.